### ISSUE TITLE
cmake error: missing CMakeLists.txt in caffe2/contrib/opengl

### ISSUE THREAD

**Issue by dbermond** [facebookarchive/caffe2#1066]
I'm getting the following error during cmake execution:
```
CMake Error at caffe2/contrib/CMakeLists.txt:6 (add_subdirectory):
  The source directory

    /home/user/abs/caffe2-git/src/caffe2-git/caffe2/contrib/opengl

  does not contain a CMakeLists.txt file.
```

It blocks the correct cmake configuration:
```
-- Configuring incomplete, errors occurred!
```
Any simple cmake invocation will result in this error. For example, this one:
`cmake -DCMAKE_CXX_COMPILER='/usr/bin/g++-5' -DCMAKE_C_COMPILER='/usr/bin/gcc-5' ..`

A git bisect shows that this is being caused by commit 2d5f1ab3cca905518da20e790a56cccd8a9cdab4.

**System information:**
**OS:** Arch Linux x86_64
**caffe2:** git master (currently at 0.8.1.r13.g596e84b7)
**cmake:** 3.8.2

**Comment by Yangqing** [comment]
Yup, we are working on it - for now check out the files from here:

https://github.com/Yangqing/caffe2/commit/5638ffe912a924513c188563aa1a64c1d3c11e0c

**Comment by dbermond** [comment]
Thank you for the attention.

**Comment by Yangqing** [comment]
Fixed in f0ea8046c26d9c324dc34deb2bfd8e3738e9f02c
